src/tests/tests.rs

```diff
@@ -0,0 +1,14 @@
+use super::Histogram;
+
+#[path = "helpers.rs"]
```

Comment:
Apparently the compiler is picky about this as of recently; I was surprised too. In future versions it will be a hard error, and right now it's a warning (which this project's setup makes an error anyway): https://github.com/rust-lang/rust/issues/37872